### PR TITLE
Remove legacy skin

### PR DESCRIPTION
**Pull request by stonko1994**

### CODE REVIEW

File: src/ts/uifactory.ts [hunk 1]
```diff
@@ -41,7 +41,6 @@ import { MetadataLabel, MetadataLabelContent } from './components/metadatalabel'
 import { PlayerUtils } from './playerutils';
 import { Label } from './components/label';
 import { CastUIContainer } from './components/castuicontainer';
-import { VolumeControlButton } from './components/volumecontrolbutton';
```
Comment (stonko1994) [inline src/ts/uifactory.ts]: @protyposis can we remove the `VolumeControlButton` at all?

Comment (protyposis) [reply src/ts/uifactory.ts]: Let's keep it, maybe somebody can make use of it.

File: CHANGELOG.md [hunk 2]
```diff
@@ -18,6 +18,7 @@ and this project adheres to [Semantic Versioning](http://semver.org/).
 
 ### Removed
 - `SubtitleSettingsPanel` in favour of `SubtitleSettingsPanelPage` to use with the new navigation feature of `SettingsPanel`
+- All variants of `skin-legacy`
```
Comment (protyposis) [inline CHANGELOG.md]: `` Player v6 legacy skin (`skin-legacy`) and its UI variants``

File: src/scss/skin-modern/_skin.scss [hunk 3]
```diff
@@ -1,94 +1,93 @@
...
   user-select: none;
+}
 
-  &.#{$prefix}-player-state-idle {
-    .#{$prefix}-ui-controlbar,
-    .#{$prefix}-ui-titlebar,
-    .#{$prefix}-ui-hugeplaybacktogglebutton {
-      display: none;
-    }
+&.#{$prefix}-player-state-idle {
```
Comment (protyposis) [inline src/scss/skin-modern/_skin.scss]: I think you need to nest all into `.#{$prefix}-ui-skin-modern {`, else the `&` extender does not really work as there is no parent?

Comment (stonko1994) [reply src/scss/skin-modern/_skin.scss]: FYI: It actually behaves like the `&` isn't there. 

It looks a bit weird so I will change it anyway.

**Pull request merged**